regression/contracts/history-pointer-enforce-03/main.c

```diff
@@ -0,0 +1,14 @@
+void foo(int *x) __CPROVER_assigns(*x)
+  __CPROVER_requires(*x > 0 && *x < __INT_MAX__ - 5) __CPROVER_ensures(
```

Comment:
I have fixed the `INT_MAX` issue. However, regarding the placement of `__CPROVER_ensures`, the current location is what clang is proposing. I will try what you suggested, but it might not work.

Reply:
Oh I see. Yeah `clang-format` goes crazy on contract annotations sometimes.

I usually surround contracts with `/* clang-format off */` and `/* clang-format on */`.

Reply:
This positioning of `_CPROVER_ensures` is what clang is proposing. I will try to move it ot the next line, but I might be unable to do this.